scripts/do-release.sh

```diff
@@ -81,13 +89,15 @@ function push_to_repo() {
 function create_pr() {
     local branch="$1"
     local msg="$2"
-    local base_branch="${release['branch']:-devel}"
+    local title="$3"
```

Comment:
```suggestion
    local title="${3:-${msg}}"
```

To preserve existing code behavior